en/README.md

```diff
@@ -56,9 +54,10 @@ If you have specific questions that are not answered by the documentation, these
 Use Github for bug reports/enhancement requests:
 
 * [C++ API](https://github.com/mavlink/MAVSDK/issues)
-* [C++ Documentation](https://github.com/dronecore/sdk_docs/issues)

```

Comment:
Haha, that was even before DronecodeSDK!

Reply:
Yeah, I may have been a bit slack on maintenance :-0